doc/nrf/libraries/networking/aws_iot.rst

```diff
@@ -18,25 +18,32 @@ For more information regarding AWS FOTA, see the documentation for the :ref:`lib
 .. note::
    The AWS IoT library and the :ref:`lib_aws_fota` library share common steps related to `AWS IoT console`_ management.
 
+Requirements
+************
+
+This library is meant for use with the :ref:`AWS FOTA <aws_fota_sample>` sample available for the nRF9160 DK.
+As part building the sample, you :ref:`create a thing in AWS IoT <creating_a_thing_in_AWS_IoT>` and generate certificates for the TLS connection.
```

Comment:
I am thinking now if this should be part of just the section above and not a requirement.
```suggestion
The :ref:`aws_iot` sample available for the nRF9160 DK showcases the use of this library.
As part of building the sample, you :ref:`create a thing in AWS IoT <creating_a_thing_in_AWS_IoT>` and generate certificates for the TLS connection.
```